code/game/machinery/rotating_alarm.dm

```diff
@@ -41,6 +41,7 @@
 	active_power_usage = 0
 	anchored = TRUE
 
+	obj_flags = OBJ_FLAG_ANCHORABLE | OBJ_FLAG_ROTATABLE
```

Comment:
I am slightly confused here..? Why would you make it anchorable?

Reply:
If you want to make it so people can build/destroy them - make them operate like light fixtures that can only fit on the walls.

Reply:
ok i fixed it (i got rid of it entirely)